app/_src/mesh/guides/jwt-validation.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Use OPA to validate JWT tokens
+content-type: tutorial
+description: Validate JWT tokens using an OPA policy in Kong Mesh.
+---
+
+A companies commonly face the following security problems:
+* How to determine that a user is who they say they are when logging in
+* How to ensure that only authorized users have access to certain applications and enforce security policies over API requests and for upstream services. 
+
+These problems can be solved by using a {{site.mesh_product_name}} [MeshOPA (beta)](/mesh/{{page.kong_version}}/features/meshopa/) policy to authorize JWT tokens by using [Envoy's External authorization filter](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ext_authz_filter.html) with [OPA](https://www.openpolicyagent.org/docs/latest/envoy-introduction/). Keep in mind that OPA doesn't handle authentication, it only handles authorization.
+
+This guide explains how to configure and deploy a MeshOPA policy that authorizes users based on if they have a valid JWT token. <!--what are the advantages of this over other options?-->.
+
+{diagram that explains how the tech works}
+
+## Prerequisites
+
+* [Deploy {{site.mesh_product_name}}](/mesh/{{page.kong_version}}/production/deployment/) in either standalone or multi-zone mode.
+* A valid JWT token stored securely?
+* install example app?:
+    ```bash
+    kubectl apply -f https://bit.ly/demokuma
+    ```
+    why? what does this contain? maybe not since we want this to apply to a bunch of use cases?
```

Comment:
@lahabana As I was writing this, I wasn't sure if we'd want to include a demo app/tutorial aspects to this. Since that's already kind of covered in the MeshOPA docs, I figured this would be best as a "here's exactly what you need to deploy this and here's a template you can copy/paste in your own values and we'll help you understand what the template is doing"